omero/sysadmins/limitations.rst

```diff
@@ -79,6 +83,26 @@ This primarily affects the following file formats:
 This issue can be avoided by pre-generating pyramidal OME-TIFF images as
 described above.
 
+
+.. _floatoverride-limitation:
+
+Protection from resource intensive minima/maxima calculations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:model_doc:`Pyramids <omero-pyramid/>` of sub-resolution image tiles are not generated for images with floating point pixels. 
+However, in some cases, these images can be viewed in OMERO clients at full resolution (if the images are not too large).
```

Comment:
Maybe remove that heading above (since we are still on the subject of `Large images with floating-point pixel data types`. And remove the 1st sentence of this paragraph, since it repeats what's above.
Then: "However, in some cases, large floating-point images without pyramids can be viewed..."